### PR TITLE
docs: Update tmpfs partition guidance to include swap

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

**Comment by mvines** [comment]
I updated the cluster scripts to match: https://github.com/solana-labs/cluster/commit/2a516e448a2edee12b638a73b79bfe0fc6d34e13

File: docs/src/running-validator/validator-start.md [hunk 1]
```diff
@@ -371,17 +371,35 @@ Once your validator is operating normally, you can reduce the time it takes to
...
+partition.
+
+Example configuration:
+1. `sudo mkdir /mnt/solana-accounts`
+2. Add a 300GB tmpfs parition by adding a new line containing `tmpfs
+   /mnt/solana-accounts tmpfs rw,size=300G,user=sol 0 0` to `/etc/fstab`
+   (assuming your validator is running under the user "sol").  **CAREFUL: If you
+   incorrectly edit /etc/fstab your machine may no longer boot**
+3. Create a 250GB swap file with `sudo fallocate -l 250G /swapfile`, `sudo mkswap /swapfile`, `sudo chmod 0600 /swapfile`
+4. Add the swap file to `/etc/fstab` with a new line containing `/swapfile swap swap defaults 0 0`
```
Comment (t-nelson) [inline docs/src/running-validator/validator-start.md:400]:
```suggestion
3. Create at least 250GB of swap space
  - Choose a device to use in place of `SWAPDEV` for the remainder of these instructions.  Ideally select a free disk partition of 250GB or greater on a fast disk. If one is not available, create a swap file with `sudo fallocate -l 250G /swapfile`, set its permissions with `sudo chmod 0600 /swapfile` and use `/swapfile` as `SWAPDEV` for the remainder of these instructions
  - Format the device for usage as swap with `sudo mkswap SWAPDEV`
4. Add the swap file to `/etc/fstab` with a new line containing `SWAPDEV swap swap defaults 0 0`
```

**Review by t-nelson — APPROVED** [review]
Ah, you beat me to it!  Thanks!  LGTM.  Just one suggestion adding clarity around swap device selection

**Comment by t-nelson** [comment]
I think the 1.4 release just clobbered this change for the published docs

**Comment by CriesofCarrots** [comment]
> I think the 1.4 release just clobbered this change for the published docs

Ugh, I thought the docs were only supposed to be published off latest beta release?

**Comment by t-nelson** [comment]
Me too! :confused: 

I guess they do what the want

**Pull request merged**